DESCRIPTION

```diff
@@ -13,7 +14,10 @@ Imports:
     shiny,
     shinyGovstyle,
     shinyjs,
-    styler
+    styler,
+    RCurl,
+    stringr,
+    htmltools
```

Comment:
Looking at the warning in the R CMD check - seems like we might need to declare 'checkmate' and 'glue' explicitly in here, can do that by:

```
usethis::use_package("checkmate")
usethis::use_package("glue")
```